src/Auth.php

```diff
@@ -2,28 +2,32 @@
 
 class Auth
 {
-    const URL = 'https://authenticate.teamworkpm.net/';
+    private static $url = 'https://authenticate.teamwork.com/';
```

Comment:
could you explain why you changed that to `private static`?

Reply:
The variable is no longer constant, the url changes for European teamwork instances to https://authenticate.eu.teamwork.com/

https://github.com/loduis/teamwork.com-project-management/pull/44/files#diff-183bd362da85e526f0f8ffc30e9c8e4cR46